.github/workflows/ci.yml

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  macos_debug_build_test:
+    runs-on: macos-10.15
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Switch to Xcode 11.2.1
+      run: sudo xcode-select --switch /Applications/Xcode_11.2.1.app/Contents/Developer
+    - name: macOS Debug Build and Test
+      run: |
+        source scripts/xcodebuild_wrapper.sh
+        macos_build OfficeUIFabricTestApp Debug build test
+  
+  macos_release_build_test:
+    runs-on: macos-10.15
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Switch to Xcode 11.2.1
+      run: sudo xcode-select --switch /Applications/Xcode_11.2.1.app/Contents/Developer
```

Comment:
Minor but could be nice to pull this out too. Then when we're on our new xcode version we only need to update it in one place and not risk forgetting about it for a flavor

Reply:
Totally agree. I'd like to pull out all of these steps and just take in the one interesting line at the bottom, but I'm not quite sure how to do that in GitHub Actions. I'll try to read up on it.

Reply:
I believe the latest iteration addresses this concern by switching to a strategy matrix and only defining these steps once.